### DISCUSSION TITLE
pcloud does  not work

### DISCUSSION THREAD

**Discussion by constantius24** [itsManjeet/avyos#97] [General]
pcloud does not work on rlxos linux. You cant login to the service

**Comment by ghost** [comment]
do you have any logs or error message you are getting?
Or it might be #56

**Reply by itsManjeet** [reply]
No logs or any output from there binary, just a timeout message after a minute from signup or login. Might me linked to #56. We'll look at it after fixing #56

**Reply by itsManjeet** [reply]
FIXED with #56
